master/buildbot/test/unit/test_status_github.py

```diff
@@ -170,11 +170,28 @@ def test_startService(self):
 
         self.status._status.subscribe.assert_called_with(self.status)
 
+    def test_stopService(self):
+        """
+        When stopped it will unsubscribe from all statuses.
+        """
+        builderstatus = FakeBuilderStatus()
+
+        result = self.status.builderAdded('builder-name', builderstatus)
```

Comment:
wait what..

There is no builderStatus in buildbot nine. I guess this was a bug on eight?
github status is not yet ported to nine, so it needs to be ported first.